api/src/pcapi/connectors/entreprise/sirene.py

```diff
@@ -30,9 +30,9 @@ def get_siret(siret: str, raise_if_non_public: bool = True) -> models.SiretInfo:
     return get_backend(settings.SIRENE_BACKEND).get_siret(siret, raise_if_non_public=raise_if_non_public)
 
 
-def siret_is_active(siret: str) -> bool:
+def siret_is_active(siret: str, raise_if_non_public: bool = True) -> bool:
     """Return whether the requested SIRET is active."""
-    siret_info = get_siret(siret)
+    siret_info = get_siret(siret, raise_if_non_public)
```

Comment:
```suggestion
    siret_info = get_siret(siret, raise_if_non_public= raise_if_non_public)
```
Vu que c'est un kwarg, autant l'utiliser comme tel